internal/pkg/aws/ecs/task.go

```diff
@@ -276,6 +307,14 @@ func TaskID(taskARN string) (string, error) {
 	return taskID, nil
 }
 
+// ShortTaskID shortens a task ID to a specified length.
+func ShortTaskID(id string) string {
```

Comment:
What do you think of this?
```suggestion
// ShortTaskID shortens a task ID to a specific length.
func ShortTaskID(id string, length int) string {
```